docs/istio-csr-external-ca-setup.md

```diff
@@ -10,23 +9,34 @@ certificates using an Issuer provided by the Trusted Certificate Service (TCS).
 ## Prerequisites for running this example:
 
   - istioctl
+  - Helm
   - Kubernetes cluster with at least one Intel SGX enabled node
-  - cert-manager
   
 ## Deployment
 
+Install `cert-manager` using Helm. You can use any other mechanism for installation as long as the end result is the same.
+
+```sh
+# Helm setup
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+
+# cert-manager install
+helm install cert-manager jetstack/cert-manager --version 1.6.1 --namespace cert-manager --create-namespace --set installCRDs=true
+```
+
 Deploy TCS and custom resource definitions (CRDs).
 
 ```sh
 kubectl apply -f deployment/crds/
 kubectl apply -f deployment/tcs_issuer.yaml
 ```
 
-Create a TCS Issuer that could sign certificates for `istio-system` namespace
+Create a TCS Issuer that could sign certificates for `istio-system` namespace. We also create the `istio-namespace` since that is where the certificates will be placed.
```

Comment:
"... also create namespace `istio-system` since ..."

Reply:
Let's merge this, I will fix this in a follow-up.